app/views/user_mailer/ranking.html.haml

```diff
@@ -19,6 +18,8 @@
     #{link_to 'I am seeking a full-time software engineering position.', 'https://mitre.referrals.selectminds.com/jobs/search/506904'}
   %p
     #{link_to 'I am seeking a full-time cyber security position.', 'http://f.rfer.us/MITREvG.1q2'}
+  %p
+    Also, please take the time to make sure your #{link_to 'resume is uploaded', edit_user_registration_url} to the scoreboard as well during this step.
```

Comment:
Recommended change: "`Please take this time to make sure your #{link_to 'resume is uploaded', edit_user_registration_url}.`"

Reply:
The wording here is meant to be a bit ambiguous since the first place team will see this twice if they have competing for prizes checked. Would you be okay with leaving the "Also" due to this.